website/docs/docs/cloud/connect-data-platform/connnect-bigquery.md

```diff
@@ -52,6 +52,122 @@ As an end user, if your organization has set up BigQuery OAuth, you can link a p
+
+</Expandable>
+
+<Expandable alt_header="Retries">
+
+Retries in BigQuery help to ensure that jobs complete successfully by trying again after temporary failures, making your operations more robust and reliable.
+
+</Expandable>
+
+<Expandable alt_header="Location">
+
+The `location` of BigQuery datasets can be set using the `location` setting in a BigQuery profile. As per the [BigQuery documentation](https://cloud.google.com/bigquery/docs/locations), `location` may be either a multi-regional location (for example, `EU`, `US`), or a regional location (like `us-west2`).
+
+</Expandable>
+
+<Expandable alt_header="Maximum bytes build">
+
+When a `maximum_bytes_billed` value is configured for a BigQuery profile, that allows you to limit how much data your query can process. It’s a safeguard to prevent your query from accidentally processing more data than you expect, which could lead to higher costs. Queries executed by dbt will fail if they exceed the configured maximum bytes threshhold. This configuration should be supplied as an integer number of bytes.
+
+If your `maximum_bytes_billed` is 1000000000, you would enter that value in the `maximum_bytes_billed` field in dbt cloud.
+
+|  Config          | Description                                                             | Type    | Example |
+| ---------------- | ------------------------------------------------------------------------| ------- | -------- |
+| Max bytes billed | The max number of bytes that can be billed for a given BigQuery query   | Integer | `1000000000` |
+
+</Expandable>
+
+<Expandable alt_header="Execution project">
+
+By default, dbt will use the specified `project`/`database` as both:
+
+1. The location to materialize resources (models, seeds, snapshots, and so on), unless they specify a custom project/database config
+2. The GCP project that receives the bill for query costs or slot usage
+
+Optionally, you may specify an execution project to bill for query execution, instead of the project/database where you materialize most resources.
+
+</Expandable>
+
+<Expandable alt_header="Impersonate service account">
+
+This feature allows users authenticating using local OAuth to access BigQuery resources based on the permissions of a service account.
+
+For a general overview of this process, see the official docs for [Creating Short-lived Service Account Credentials](https://cloud.google.com/iam/docs/create-short-lived-credentials-direct).
+
+</Expandable>
+
+<Expandable alt_header="Job retry deadline seconds">
+
+Job retry deadline seconds is the maximum amount of time BigQuery will spend retrying a job before it gives up.
+
+</Expandable>
+
+<Expandable alt_header="Job creation timeout seconds">
+
+Job creation timeout seconds is the maximum time BigQuery will wait to start the job. If the job doesn’t start within that time, it times out.
+
+</Expandable>
+
```

Comment:
gcs buckec, dataproc cluster name, and dataproc region are for running dbt python mods on gcp. there's not mention of that here in the doc so do you think we should have a mini h4 header separating the remainder of the options telling users that the following are for that? we do mention it in the core docs: https://docs.getdbt.com/docs/core/connect-data-platform/bigquery-setup#running-python-models-on-dataproc